agent/Cargo.toml

```diff
@@ -12,6 +12,8 @@ async-trait = "0.1.0"
 blake2 = "0.8.0"
 chrono = "0.4.10"
 cfg-if = "0.1"
+coap = { git = "https://github.com/jiayihu/coap-rs", branch = "receive_from", optional = true }
```

Comment:
I'll remove my own fork when the PR https://github.com/Covertness/coap-rs/pull/60 is merged